load_test/lib/load_test/user/sse.ex

```diff
@@ -32,10 +37,17 @@ defmodule SseUser do
     signed = JOSE.JWT.sign(context.sse_jwt_secret, jws, jwt)
     {%{alg: :jose_jws_alg_hmac}, compact_signed} = JOSE.JWS.compact(signed)
 
+    last_event_id =
```

Comment:
**Suggestion** 🤔 : this can be done by using pattern matching too

```
defp last_event_id_header(nil), do: []
defp last_event_id_header(last_event_id), do: [{"Last-Event-ID", last_event_id}]
```

and then

```
    [
      {["Authorization"], "Bearer #{compact_signed}"},
      {["User-Agent"], context.sse_user_agent}
    ] ++ last_event_id_header(state.last_event_id)
```

Reply:
Done